### PR TITLE
Ensure that README code example is up to date

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
This provide a simple way to check that your README example is up-to-date.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -306,6 +306,13 @@ extern crate unicode_xid;
 #[cfg(feature = "printing")]
 extern crate quote;
 
+#[cfg(doctest)]
```
Comment (taiki-e) [inline src/lib.rs:309]: Note that this increases MSRV to 1.40. https://github.com/BurntSushi/termcolor/issues/35

Comment (GuillaumeGomez) [reply src/lib.rs:309]: Outch, good point!

**Review by dtolnay — COMMENTED** [review]
I would prefer not to make this change. You've sent equivalent PRs to some of my other projects in the past, which were similarly rejected. Could you please avoid sending any more of these to all projects under my account?

**Comment by GuillaumeGomez** [comment]
Sorry, was using syn in one of my project and realized that, completely forgot I already sent the same change proposal to other project of yours. Will try to not do it again.

**Pull request closed without merging**